tests/notification/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM python:3-slim
+
+ENV POETRY_HOME="/opt/poetry" \
+    POETRY_VIRTUALENVS_IN_PROJECT=1 \
+    POETRY_NO_INTERACTION=1 \
+    GECKODRIVER="0.35.0"
+
+ENV PATH="$POETRY_HOME/bin:$PATH"
+
+ENV ENV="dev"
```

Comment:
nit: 'ENV' is very generic, perhaps `NOTIFICATION_TEST_ENV` or `TEST_ENV` instead?